src/extension/ncwms/pom.xml

```diff
@@ -64,6 +64,17 @@
             <artifactId>junit</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.httpcomponents</groupId>
+            <artifactId>httpclient</artifactId>
+            <version>4.5.1</version>
```

Comment:
Is it necessary to include the version? 

I'm sure Geoserver is going to include this dep as well. How will the two build systems resolve which version gets included?